### ISSUE TITLE
Using activation whitelist prevents addon from functioning

### ISSUE THREAD

**Issue by samamstar** [spenibus/cors-everywhere-firefox-addon#57]
### Issue:
Whenever I set any filters, the extension fails to function

### System info:
| Firefox version | 116.0.3 |
| --- | --- |
| Addon version | 18.11.13.2043 |
| OS | Windows 11 22621.2134 (22H2) |
| Addon settings | Enabled on startup TRUE</br>Force acao EMPTY</br>Filter SEE BELOW |

### Diagnosis steps taken:
- Cleared filter
- Tested at http://spenibus.net/cors-everywhere-test.html
  - Readystate 2,3,4 all status 200
- Set filter to ".+"
- Tested at http://spenibus.net/cors-everywhere-test.html
  - Did not work, Readystate 4 status 0
- Restarted browser
  - No change
- Restarted computer
  - No change

### Final thoughts
I know this is the exact opposite of #56, but it's what's happening. 
My Firefox install has some issues with CORS that I'm trying to use this extension to sidestep. There's a specific website I use (https://dicecloud.com) that delivers fonts correctly on my desktop machine, but not this laptop. Very frustrating

edit: correctly not directly

**Comment by spenibus** [comment]
We might run into the same issue I had with #56, where I can't fix what I can't reproduce.

Just to complete this (nicely written) bug report:

1. You made sure to use `/.+/` in the filter ?
2. Have you tried running the addon with a new profile containing no other addons ?

I'm still running win10 but I doubt that's the issue, although we never know...

Had a sudden idea... I managed to break the whitelist if I insert at least one empty line before the first filter. But that's all I got so far.
